plugins/org.jkiss.dbeaver.ui.editors.connection/src/org/jkiss/dbeaver/ui/dialogs/driver/EditMavenArtifactDialog.java

```diff
@@ -361,6 +363,11 @@ public void saxText(SAXReader reader, String data) {
 
     }
 
+    private enum TabData{
+        RAW,
+        MANUAL
```

Comment:
We don't use these terms in this dialogue anymore. After some time, we will forget them and will be pretty confused when reading the code that uses these constants. Please consider renaming.